ONSdigital/eq-survey-runner
# Move front end assets into "app/assets"

### What is the context of this PR?

Following our design review, the team decided to move the sources for the sits static assets into a containing folder.  This PR makes that change by collecting the static asset sources and moving them into the `app/assets` directory.
### How to review

1) Checkout the branch
2) Run `npm run compile`
3) Run `npm run test`
4) Run the app and ensure everything looks the way it should
### Who worked on the PR

@weapdiv-david made these changes, they should probably be reviewed by @hamishtaplin or @bennpearson
